src/runtime_src/core/pcie/tools/cloud-daemon/msd.cpp

```diff
@@ -286,27 +287,45 @@ static void msd(std::shared_ptr<pcidev::pci_device> d, std::string host)
             }
         }
 
+		retLocal = retRemote = -100;
         // Waiting for msg to show up, interval is 3 seconds.
-        ret = waitForMsg(dev, mbxfd, mpdfd, 3);
+        ret = waitForMsg(dev, mbxfd, mpdfd, 3, retLocal, retRemote);
```

Comment:
please use a 2-member array for less code duplication.

Reply:
will change